src/utils/workflow_utils/src/workflow_utils.c

```diff
@@ -1804,6 +1816,107 @@ bool workflow_set_sandbox(ADUC_WorkflowHandle handle, const char* sandbox)
     return true;
 }
 
+bool _workflow_cpy_config_download_folder(char* dest, const size_t dest_size)
+{
+    bool success = false;
+    const ADUC_ConfigInfo* config = ADUC_ConfigInfo_GetInstance();
+    if (config == NULL)
+    {
+        goto done; // could not get
+    }
+
+    size_t download_folder_len = ADUC_StrNLen(config->downloadsFolder, dest_size);
+
+    if (download_folder_len == 0 || download_folder_len == dest_size)
+    {
+        Log_Error("Invalid download folder length for config");
+        goto done;
+    }
+
+    if (ADUC_Safe_StrCopyN(dest, config->downloadsFolder, dest_size, download_folder_len) != download_folder_len)
+    {
+        Log_Error("Failed copy config download folder");
+        goto done;
+    }
+
+    success = true;
+done:
+    ADUC_ConfigInfo_ReleaseInstance(config);
```

Comment:
ReleaseInstnace? Presumably GetInstance did locking or a refcount on the configinfo?